components/molecules/NivoScatterChart/nivo-scatter-chart.tsx

```diff
@@ -72,10 +101,30 @@ const NivoScatterPlot = ({
 
   return (
     <>
-      <div className="flex flex-col md:flex-row justify-between px-7 pt-5">
+      <div className="flex flex-col md:flex-row justify-between px-7 pt-3 items-center">
         <Title level={4} className="!text-sm  !text-light-slate-12">
           {title}
         </Title>
+        <ToggleGroup handleChange={handleTogglePrFilter} className="hidden lg:flex">
+          <>
```

Comment:
Can you please create a loop to avoid duplicate code 

`{YourDataStructures.map(data : ... => {`
` return <div> {data.name} </div> <span className="ml-2 py-`
`0.5 px-1.5 h-fit bg-slate-200 text-slate-500 border rounded-full ``text-xs font-semibold"> `
`{humanizeNumber(metadata... null)}`
`</span>`

` })}`

Reply:
You are correct about the duplicate code issue, but that wouldn't be the correct way to fix it. This would just make the code more complicated, as new objects will need to be created just for the reason to be dynamically iterated through. In case of just 3 options, that's not worth it, especially given that the code will be harder to read developer-wise.

The better solution for this would be to make a new component for this
```ts
<span className="ml-2 py-
0.5 px-1.5 h-fit bg-slate-200 text-slate-500 border rounded-full ``text-xs font-semibold"> 
...
</span>
```
since that's mainly the repeated part, but that's something for another issue.